FAILED UkrNet/tests/test_login_page.py::TestLoginPage::test_login_with_existent_credentials - AssertionError: Inbox isn't opened
FAILED UkrNet/tests/test_login_page.py::TestLoginPage::test_login_with_username_inside_spaces - AssertionError: Inbox isn't opened





docker run --privileged -it [image]

docker run --privileged -it -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix sha256:5c58fa84af9dd01fd9df9748a84f5fb270439db5286da20afcbd2f486801ff4b
